action.yml

```diff
@@ -1,7 +1,10 @@
 name: "Pantheon deploy"
-description: "This GitHub Action deploys your current issue branch to a remote Pantheon repo."
+description: "NodeJS action to deploy a branch to Pantheonfor FAST deployment with rsync/ssh"
 author: "irishdistillers"
 inputs:
```

Comment:
Missing other inputs E.g: STRICT_BRANCH_NAMES, PR_STATE ...

Reply:
What is going on with my local branch resetting files even though I update them all the time ... Fixing